python/pyspark/ml/regression.py

```diff
@@ -180,10 +180,7 @@ class LinearRegression(_JavaRegressor, _LinearRegressionParams, JavaMLWritable,
     >>> test1 = spark.createDataFrame([(Vectors.sparse(1, [0], [1.0]),)], ["features"])
     >>> abs(model.transform(test1).head().newPrediction - 1.0) < 0.001
     True
-    >>> lr.setParams("vector")
-    Traceback (most recent call last):
-        ...
-    TypeError: Method setParams forces keyword arguments.
+    >>> lr.setParams(featuresCol="vector")
```

Comment:
@HyukjinKwon is it correct here just to specify no output from this command? I don't think we care about testing any output of this. I don't know how python docs work though :)

Reply:
Hi, @srowen . +1 for removing this ancient `TypeError` example (added 5 year ago). In this case, we can do partial check like the following.
```python
>>> lr.setParams(featuresCol="vector")
LinearRegression...
```

Reply:
Yeah ^^ seems correct